components/developer/clang-80/clang-80.p5m

```diff
@@ -49,6 +49,7 @@ link path=usr/bin/clang-offload-bundler target=../clang/8.0/bin/clang-offload-bu
 link path=usr/bin/clang-refactor target=../clang/8.0/bin/clang-refactor mediator=clang mediator-version=8.0
 link path=usr/bin/clang-rename target=../clang/8.0/bin/clang-rename mediator=clang mediator-version=8.0
 link path=usr/bin/clang++ target=../clang/8.0/bin/clang++ mediator=clang mediator-version=8.0
+link path=usr/bin/clang++-8 target=../clang/8.0/bin/clang++-8 mediator=clang mediator-version=8.0
```

Comment:
Do we need mediator here?

Reply:
I think, absolutely not (see the discussion in the PR) :) 

But as posted here, it follows the precedent set by existing (and IMHO flawed) p5m, where `/usr/bin/clang-8` is also mediated (IMHO needlessly).

Transactionally speaking, I think it makes sense to address (drop) the mediation of numbered-suffix names, but it belongs in a different PR to not stall this small change.